tests/src/test/java/alluxio/server/ft/journal/TriggeredCheckpointTest.java

```diff
@@ -121,7 +121,7 @@ private void validateCheckpointInClusterRestart(MultiProcessCluster cluster)
       throws Exception {
     cluster.stopMasters();
     cluster.startMasters();
-    cluster.waitForAllNodesRegistered(20 * Constants.SECOND_MS);
+    cluster.waitForAllNodesRegistered(30 * Constants.SECOND_MS);
```

Comment:
why does it take so long to register nodes?

Reply:
I am not exactly sure. There might be some parameters that we can tune to make it faster.